File: in-motion-ios/inMotion/inMotion/Models/DTO/Post/Incoming/PostReactionWithoutAuthorDto.swift
//
//  POstReactionWithoutAuthorDto.swift
//  inMotion
//
//  Created by Kamil Pietrak on 02/12/2023.
//

import Foundation

struct PostReactionWithoutAuthorDto: Codable {
    public var id: UUID

    public var authorId: UUID
    public var emoji: String
    public var createdAt: Date
    
    
}

File: in-motion-server/Post/IMS.Post.Models/Dto/Outgoing/PostReactionWithoutAuthorDto.cs
namespace IMS.Post.Models.Dto.Outgoing;

public class PostReactionWithoutAuthorDto
{
    public string Id { get; set; }
    
    public string AuthorId { get; set; }
    
    public string Emoji { get; set; }
    
    public DateTime CreatedAt { get; set; }
}